src/account/AccountStorage.sol

```diff
@@ -39,6 +39,8 @@ struct SelectorData {
     // Note that even if this is set to true, user op validation will still be required, otherwise anyone could
     // drain the account of native tokens by wasting gas.
     bool isPublic;
```

Comment:
This field being `true` also indicates the function is not state-change, right? Might be worth adding it to the docs.

Reply:
Technically, it can still be state-changing, it's just that usually this will only be used for `view` functions. See this PR comment for more context: https://github.com/erc6900/reference-implementation/pull/61